students/shibata/session3/mailroom.py

```diff
@@ -0,0 +1,150 @@
+#! /usr/bin/env Python
+"""
+It should have a data structure that holds a list of your donors and a history of the amounts they have donated. This structure should be populated at first with at least five donors, with between 1 and 3 donations each.
+
+You can store that data structure in the global namespace.
+The script should prompt the user (you) to choose from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”)
+"""
+
+from time import sleep
+import pprint
+
+if __name__ == '__main__':
+    objFileName = "C:\Files\donate.txt"
+    strData = ""
+    dicRow = {}
```

Comment:
no need to initilize and empty one -- it will jsut get thrown away when you create the first real one.